cvat-cli/src/cvat_cli/parser.py

```diff
@@ -201,6 +201,22 @@ def make_cmdline_parser() -> argparse.ArgumentParser:
         action="store_true",  # automatically sets default=False
         help="""zip chunks before sending them to the server""",
     )
+    task_create_parser.add_argument(
+        "--cloud_storage_id",
+        default=None,
+        type=int,
+        help="cloud storage ID if you would like to use data from cloud storage",
+    )
+    task_create_parser.add_argument(
+        "--filename_pattern",
+        type=str,
+        help="""pattern for filtering data from the manifest file for the upload.
```

Comment:
Multi-line help is not supported by argparse by default.

```
 --filename_pattern FILENAME_PATTERN
                        pattern for filtering data from the manifest file for the upload. Only shell-style wildcards are supported: * - matches
                        everything ? - matches any single character [seq] - matches any character in 'seq' [!seq] - matches any character not in seq
```

Reply:
```
--filename_pattern FILENAME_PATTERN
                        pattern for filtering data from the manifest file for the upload.
                        Only shell-style wildcards are supported:
                        * - matches everything
                        ? - matches any single character
                        [seq] - matches any character in 'seq'
                        [!seq] - matches any character not in seq
```